README.md

```diff
@@ -2420,17 +2423,17 @@ Some notes on the negotiation process itself:
   This can put the company in a difficult position (they may have declined another key
   candidate based on your acceptance), and may hurt your reputation in unexpected ways
   later.
-- Robby Grossman, a VP at Wistia, gives
-  [a good overview](http://rob.by/2013/negotiating-your-startup-job-offer/) of equity
-  compensation and negotiation suggestions.
+- Robby Grossman, a VP at Wistia, gives a
```

Comment:
👍 on keeping links consistently on the noun sans article.

Reply:
👍